modules/apigee_edge_teams/src/Entity/Team.php

```diff
@@ -347,4 +347,14 @@ protected static function propertyToBaseFieldBlackList(): array {
     ]);
   }
 
+  /**
+   * This is required by jsonapi.
+   *
+   * This is here because the jsonapi module relies on a uuid to build end points and breaks if one is not present.
+   * Creating a real UUID for this entity type is not possible as entity data is stored on Apigee.
```

Comment:
But it is not a problem because the JSONAPI _specification_ also allows non-valid UUID as universal identifiers until their uniqueness is ensured.
https://jsonapi.org/format/#crud-creating-client-ids